Dockerfile

```
@@ -0,0 +1,48 @@
+FROM debian:12-slim AS builder
+
+RUN apt-get update
+RUN apt-get dist-upgrade -y
+
+RUN apt-get install -y build-essential libzmq3-dev cmake
+
+WORKDIR /build
+
+COPY cmake/ ./cmake/
+COPY CMakeFiles/ ./CMakeFiles/
+COPY src/ ./src/
+COPY CMakeLists.txt .
+
+RUN mkdir out && cd out && cmake .. && make
```

Can we change this to:

```suggestion
RUN mkdir out && cd out && cmake -DCMAKE_BUILD_TYPE=Release .. && make
```

Just to ensure it gets built with all optimizations enabled in RELEASE mode?

---

Done!